qtconsole/console_widget.py

```diff
@@ -342,16 +359,17 @@ def __init__(self, parent=None, **kw):
 
         action = QtWidgets.QAction('Select All', None)
         action.setEnabled(True)
-        selectall = QtGui.QKeySequence(QtGui.QKeySequence.SelectAll)
+        selectall = self.shortcut_manager.shortcut_select_all
```

Comment:
Similar to the comment above, `selectall` needs to be still a `QKeySequence` instance to be able to use `QKeySequence.matches` in the validation below this line. So this should look something like:
```suggestion
        selectall = QtGui.QKeySequence(self.shortcut_manager.shortcut_select_all)
```